Cargo.toml

```diff
@@ -28,7 +31,7 @@ cgmath = { version = "0.15", features = ["mint"] }
 derivative = "1.0"
 froggy = "0.4.4"
 genmesh = "0.5"
-gfx = "0.16"
+gfx = "0.16.3"
```

Comment:
Why?

Reply:
Because your instancing changes require it

Reply:
I'm not a fan of strict version rules in Cargo. It will use the latest available version anyway.

Reply:
I think your position is in the minority within the community then. Latest will be used on fresh checkout, if we don't check in `Cargo.lock`. But people who already have `Cargo.lock` will get breakage if you don't write the exact version in the `Cargo.toml`. Another case is - some crate that depends on you may say `gfx = "=0.16.2"` and it will try to build but will fail even after `cargo update`.

TL;DR: Cargo versioning rules are here to stay and be respected

Reply:
Also, there is a proposal (which I supported) about making `cargo publish` checking out all the **minimal** compatible versions (as opposed to latest). Think about it ;)

Reply:
Okay, you convicted me